### PR TITLE
Add blog post of IvanNik17

### PR DESCRIPTION
**Pull request by tkoyama010**
### Overview

Add blog post of @IvanNik17 as external examples.

 

### Details

- https://github.com/pyvista/pyvista/discussions/3126

### CODE REVIEW

**Review by banesullivan — DISMISSED** [review]

File: doc/make_external_gallery.py
Comment (adeak) [inline doc/make_external_gallery.py:170]: Two things come to mind:
1. It would be nice to have a combination of short title + longer description as in the existing examples (instead of copying the same for both fields), but I think the new website doesn't actually use the description field anywhere (@akaszynski  please correct me if I'm wrong)
2. Since the "title" is what gets rendered in the gallery, I'd consider trying to make these shorter. At least look at the built docs (I haven't, yet) an check that the text doesn't look weird or broken in the gallery.

File: doc/make_external_gallery.py
Comment (adeak) [inline doc/make_external_gallery.py:169]: One more, sorry: it seems to me that this nice yellow bunny gif was made with Vedo, not PyVista :( We should double-check that all 3 gifs were made with PyVista (the first two are probably OK, but I only skimmed the three posts a bit).

Comment (tkoyama010) [reply doc/make_external_gallery.py:169]: Thank you for the check. Sorry for the mistake. This is due to the fact that the word "PyVista" was on the right and "Vedo" was on the left. I should have looked at the captions more carefully.
image

Comment (tkoyama010) [reply doc/make_external_gallery.py:169]: Fixed in a1b7aee0a142558d73069415e02d3467a3abcc5f .

File: doc/make_external_gallery.py
Comment (tkoyama010) [reply doc/make_external_gallery.py:170]: How about following the format of a scientific paper and titling it "Ivan Nikolov, Apr 26, 2022", "Ivan Nikolov, May 31, 2022" and "Ivan Nikolov, Jun 16, 2022"? (I am not a scientist, so my writing style may be wrong.)

**Comment by adeak** [comment]
One more general question: do we know anything about copyright(/licensing??) of these gifs? Is it OK to just add them to our repo? Should we ask for permission?

I've found https://github.com/IvanNik17/python-3d-analysis-libraries which is very related, but I can't easily find these gifs in there, only source and data. But this is GPL-3.0 licensed. I don't even know if that would apply to images, or only source code... it's probably fine.

File: doc/make_external_gallery.py
Comment (adeak) [reply doc/make_external_gallery.py:170]: If we go that route I'd prefer to say something about the contents. So something like
1. Nikolov on Visualization Libraries
2. Nikolov on Voxelization and Point Clouds
3. Nikolov on Neighbourhood Analysis

If "Ivan Nikolov" fits too, that would be even better.

**Comment by tkoyama010** [comment]
@adeak This PR is not a rush, so we can wait for @IvanNik17's response before merging.

File: doc/make_external_gallery.py
Comment (tkoyama010) [reply doc/make_external_gallery.py]: Fixed in 659f3da288310efe95bd05662b5b5b22187d7311 .

File: doc/make_external_gallery.py
Comment (adeak) [inline doc/make_external_gallery.py:160]: Sorry, I promise this is the last comment for a while :D I misread the original title: it's "voxelize (meshes and point clouds)". So we can just use
```suggestion
        title="Ivan Nikolov on Voxelization",
```

Comment (tkoyama010) [reply doc/make_external_gallery.py:160]: You can make as many comments as you like.

**Comment by IvanNik17** [comment]
@tkoyama010 Thank you for considering my blog post. I just wanted to add that it's totally fine to add it. All images/gifs are made by me for the article, so copyright should be fine.

**Comment by tkoyama010** [comment]
@IvanNik17 Thanks! We are glad that we can add your great job to our page.

**Review by akaszynski — APPROVED** [review]

**Pull request merged**
